crates/driver/src/api/solve/dto/auction.rs

```diff
@@ -42,17 +45,20 @@ struct Order {
     valid_to: u32,
     #[serde_as(as = "serialize::Hex")]
     app_data: [u8; 32],
-    #[serde_as(as = "serialize::U256")]
-    fee_amount: U256,
     kind: OrderKind,
     partially_fillable: bool,
+
+    #[serde_as(as = "serialize::U256")]
+    fee_amount: U256,
+    #[serde_as(as = "serialize::U256")]
+    #[serde(default)]
+    full_fee_amount: U256,
```

Comment:
Should this be:
```suggestion
    scoring_fee_amount: U256,
```

Or something to indicate that this is the fee amount that we use for scoring?

Reply:
Oops! This is the dto 🤦